packages/core/admin/server/src/routes/admin.ts

```diff
@@ -66,4 +66,10 @@ export default [
       ],
     },
   },
+  {
+    method: 'GET',
+    path: '/cloud-user-info',
+    handler: 'admin.getCloudAuthInfo',
+    config: { auth: false },
```

Comment:
I'm a bit concerned by this. Does this mean anyone with the admin URL can get access to the cloud CLI data? :thinking:

Reply:
Yeah it would be a public endpoint but only return the info of the user who authenticated through the cloud CLI